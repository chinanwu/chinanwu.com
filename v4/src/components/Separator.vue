<template>
  <div class="Separator__container" role="presentation">
    <div class="Separator"></div>
  </div>
</template>

<script>
export default {
  name: "Separator"
};
</script>

<style lang="less" scoped>
.Separator__container {
  width: 100%;
  display: flex;
  justify-content: center;
}

.Separator {
  margin: 3rem;
  border-bottom: 2px solid #f2f2f2;
  width: 60%;
}

@media only screen and (max-width: 375px) {
  .Separator {
    width: 80%;
  }
}
</style>
