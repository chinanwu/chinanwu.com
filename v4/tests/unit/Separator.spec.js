import { shallowMount } from "@vue/test-utils";
import Separator from "@/components/Separator.vue";

describe("Separator.vue", () => {
  it("renders", () => {
    const wrapper = shallowMount(Separator);
    expect(wrapper.element).toMatchSnapshot();
  });
});
