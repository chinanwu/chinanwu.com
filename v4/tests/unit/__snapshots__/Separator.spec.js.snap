// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator.vue renders 1`] = `
<div
  class="Separator__container"
  role="presentation"
>
  <div
    class="Separator"
  />
</div>
`;
